src/managers/config.py

```diff
@@ -633,6 +633,8 @@ def cruise_control_properties(self) -> list[str]:
                 "sasl.mechanism=SCRAM-SHA-512",
                 f"security.protocol={self.security_protocol}",
                 f"capacity.config.file={self.workload.paths.capacity_jbod_json}",
+                "webserver.security.enable=true",
+                f"webserver.auth.credentials.file={self.workload.paths.cruise_control_auth}",
```

Comment:
**thought:** Ah, so it does.

Reply:
Yup, here is the [section](https://github.com/linkedin/cruise-control/wiki/Security#http-basic) for future reference